gatsby-config.js

```diff
@@ -49,10 +49,7 @@ const plugins = [
     },
   },
   {
-    resolve: `gatsby-transformer-gitinfo`,
-    options: {
-      include: /\.mdx?$/i, // Only .md files
```

Comment:
@mildronize  I think the comment is wrong. 
It is support both .md and .mdx.

If it does not support mdx, it is possible because of a bug in the plugin.
![image](https://user-images.githubusercontent.com/2938310/128169406-bd512ad6-a6b3-47fd-963b-a4a0e95f3a7b.png)

Please keep include and correct a comment.
```js
 resolve: `gatsby-transformer-gitinfo`,
    options: {
      include: /\.mdx?$/i,  // Support  both .md and .mdx files
   }

```

Reply:
Nope the existing config is wrong

Reply:
@mildronize From this document, I don't understand why it is wrong. Please explain me. Thanks.
https://www.gatsbyjs.com/plugins/gatsby-transformer-gitinfo/#configuration-options

Reply:
@mildronize @aaronamm I think the config should be /\.m(d|dx)$/i for support both file md and mdx because /\.mdx?$/i will support only mdx file